product_expiry_configurable/models/stock_production_lot.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import datetime
+
+from odoo import api, fields, models
+
+
+class StockProductionLot(models.Model):
+
+    _inherit = "stock.production.lot"
+
+    product_qty = fields.Float(store=True)
```

Comment:
Why do you need this stored?

Reply:
I was searching among lots using this field, but it is true that it does not have much sense. I remove it. Thank you!